frontend/src/component/onboarding/sdkSnippets.ts

```diff
@@ -29,12 +29,13 @@ dotnet add package Newtonsoft.Json`,
 };
 
 export const initializeCodeSnippets: Record<SdkName, string> = {
-    Node: `import { initialize } from 'unleash-client';
+    Node: `const { initialize } = require('unleash-client');
```

Comment:
This is what we suggest in out sdk github page and this is what works out of the box.

Reply:
I thought that most people use ESM modules nowadays so wanted to provide the most up to date example

Reply:
I am still not sure, because default package json is not ESM.

Reply:
If you know that you have configured type module, then you know how to change the import.
But if you have no idea, about ESM vs commonjs, you will be behind a hurdle.